examples/cryptocurrency-advanced/backend/build.rs

```diff
@@ -0,0 +1,14 @@
+extern crate protoc_rust;
+
+use protoc_rust::Customize;
+
+fn main() {
+    protoc_rust::run(protoc_rust::Args {
+        out_dir: "src/proto",
```

Comment:
(Semi) off-topic: there may be some problems pushing generated code to `src`; please see [Cargo docs for the workaround](https://doc.rust-lang.org/cargo/reference/build-scripts.html#case-study-code-generation).